### PR TITLE
Script for coverage when AutoCompleteList have values in upper bound

### PR DESCRIPTION
**Pull request by Aleksandriys**
@aarotar @OVikhrov please review.

### CODE REVIEW

File: test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      {
+        keyboardLayout = "QWERTY",
+        keypressMode = "SINGLE_KEYPRESS",
+        limitedCharacterList =
+        {
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList ={"TestList_100","TestList_100","TestList_100","TestList_100","TestList_100",
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua]: @Aleksandriys, plase do not use such big constructions in requests in test like autoCompleteList there.
You may create autoCompleteList array with needed size in --[[ Local variables ]] using, for example:

autoCompleteList ={}
for i=1, 'upperBoundValue' do
  autoCompleteList[i]=<array_element>
end

and after that use it in requests.

**Comment by Aleksandriys** [comment]
@aarotar please review.

**Comment by Aleksandriys** [comment]
@aarotar please review.

File: test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+-- [SetGlobalProperties] Conditions for SDL must transfer request to HMI in case of "autoCompleteList" param have value is UpperBound
+--
+-- Description:
+-- Case when SDL tranfer SetGlobalProperties_request with <autoCompleteList> param to HMI
+-- respond with <success = true, resultCode = "SUCCESS"> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList>  string in array is upper bound.
+-- 3. HMI respond.
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua]: @Aleksandriys, step 3. is informative, please remove it at all.

File: test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      {
+        keyboardLayout = "QWERTY",
+        keypressMode = "SINGLE_KEYPRESS",
+        limitedCharacterList =
+        {
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList = {"upperbound", "upperboundupperboundupperboundupperboundupperbound"}
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua]: @Aleksandriys, due to "[Mobile API] SetGlobalProperties request/response" non-functional req, maxlength=50.
You use 51 in script, so, actually, you check **out upper bound** case, which is not applicable.
Please update script to check upper bound value, not out upper bound.

File: test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_AutoCompleteList_String_in_Array_Is_UpperBound()
+--mobile side: sending SetGlobalProperties request
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua]: @Aleksandriys, we should not use comments if they are obvious and duplicating code itself, like there:
```
--mobile side: sending SetGlobalProperties request
local cid = self.mobileSession:SendRPC("SetGlobalProperties".
```
Please apply to another comments in script if they redundant.

**Pull request merged**